docs/developer/advanced/running-elasticsearch.asciidoc

```diff
@@ -13,7 +13,7 @@ This will run a snapshot of {es} that is usually built nightly. Read more about
 ----
 yarn es snapshot
 ----
-
+The default credentials are `elastic:changeme` (superuser) and `kibana_system:changeme` when connecting from Kibana.
```

Comment:
I think some additional context could be helpful about the difference between the two and when to use which.  

```suggestion
By default, two users are added to Elasticsearch:
  - A superuser with username: `elastic` and password: `changeme`, which can be used to log into Kibana with.
  - A user with username: `kibana_system` and password `changeme`. This is what the Kibana system uses to access certain capabilities, even when the logged in user lacks access. These credentials aren't meant to be used by a real user. They can be configured via the kibana.yml file. Read more: <<using-kibana-with-security>>.
```

@legrego - is what I wrote accurate?

Just a suggestion!

Reply:
Thanks for the suggestions both! Which suggestion should I go with? :)

Reply:
@legregos ! :D

Reply:
Done!

Reply:
The text "A user with username: kibana_system and password changeme." seems like it's missing something. Shouldn't it be "A user with username: kibana_system and password changeme exists" or "The credentials for connecting to Elasticsearch with Kibana are `kibana_system` / `changeme`"

Reply:
I think it makes sense grammatically because it's a bullet point following the top sentence.

```
By default, two users are added to Elasticsearch:
  - A superuser.
  - A Kibana system user.
```

Larry's suggestion was just to change the text of the second bullet point (I believe anyway).

```
By default, two users are added to Elasticsearch:
  - A superuser with username: `elastic` and password: `changeme`, which can be used to log into Kibana with.
  - A user with username: `kibana_system` and password `changeme`. This account is used by the Kibana server to authenticate itself to Elasticsearch, and to perform certain actions on behalf of the end user. These credentials should be specified in your kibana.yml as described in <<using-kibana-with-security>>
```

Reply:
Ahh, now it clicks. Thank you! :)